tests/php/ORM/ArrayListTest.php

```diff
@@ -133,6 +134,19 @@ public function testLimit()
         );
     }
 
+    /**
+     * @expectedException PHPUnit_Framework_Error
```

Comment:
A note for anyone who merged up this pull request - in PHPUnit 7 the equivalent of this class is `\PHPUnit\Framework\Error\Error` and it must be a FCQN, not an imported class when using it in `@expectedException`